wp-forms-api/class-wp-forms-api.php

```diff
@@ -505,13 +506,26 @@ static function render_element( $element, &$values, $form = null ) {
 				break;
 
 			case 'mce':
+				if ( ! user_can_richedit() ) {
```

Comment:
Can we do `if(` without the space so it's at least internally consistent throughout this file?